docs/api/structures/desktop-capturer-source.md

```diff
@@ -0,0 +1,14 @@
+# Desktop Capturer Source Object
```

Comment:
Should this be "DesktopCapturerSource Object"?

I don't feel strongly one way or the other, but I think so far we've mostly been CamelCasing them (or whatever all-leading-caps case is called 🤔 ).